### PR TITLE
More Realistic Error Rates

### PR DESCRIPTION
**Pull request by gordonkoehn**
Upon the feedback in the mid-term presentation
- #145 
I added another error rate condition called "Moderate" with a better ration of FPR / FNR. 

Screenshot 2023-08-15 at 16 04 07

**Is this a good set of conditions ?**

Further, I allow to select for error rate conditions in the snakemake experiments now.

### CODE REVIEW

**Comment by pawel-czyz** [comment]
I recall that the extreme FPR is 1%. Perhaps we can do three realistic settings:
```
Ideal: FPR=1e-6, FNR=1e-6
Typical: FPR=1e-6, FNR=0.1
Large: FPR=0.01, FNR=0.2
```
and for selected experiments (but not all) the `Very Very Extreme` (which is not realistic), with e.g., `FPR=FNR=0.45` to just see how quickly the MCMC explores the space in the limit when likelihood is almost non-informative.

File: workflows/mark01.smk [hunk 1]
```diff
@@ -25,10 +25,15 @@ metrics = ["MP3","AD","DL"]  # <-- configure distances here
 num_samples = 30 # <-- configure number of samples here
 
 # Errors <--- set the error rates here
-errors = {
+selected_error_cond = ['IDEAL', 'TYPICAL','MODERATE', 'LARGE']
```
Comment (pawel-czyz) [inline workflows/mark01.smk:28]: I like this parametrization of rules very much!

**Comment by gordonkoehn** [comment]
I like having fewer more realistic error combinations! :) So let's go for the 4 you propose, with one adjustment - if you agree.

where do you have the 1% FRP from - can't find a reference for that.
 
I went through all references from the mid-term presentation and chats with Johannes and believe we should use the hardest realistic setting as FPR=1e-5 / 1e-4, still 2-3 orders of magnitude over the median somatic mutation rate:

`
Ideal: FPR=1e-6, FNR=1e-6
Typical: FPR=1e-6, FNR=0.1
Large: FPR=1e-4, FNR=0.2

Extreme: FPR=0.45, FPR=0.45
`

Here is what Johannes told me a while back:

>  - Die False-Discovery Rates wurden etwas systematischer in folgendem Paper studiert:
https://www.sciencedirect.com/science/article/pii/S0092867412002280?via%3Dihub#mmc1 (Document S1 ist glaube ich aufschlussreich). Die niedrigste ADO rate die man so erwartet liegt bei etwa 7 %.

>  - In meinen (relativ modernen) Daten liegt die mittlere ADO rate abhängig vom sample bei ca. 12-16%

> - Die False-Positive rate liegt etwa in der Range von 10**-6 bis 10 **-5 und die ADO rate wird auf mehr als 25 % geschätzt in den extremsten fällen (das findest du hier: https://www.nature.com/articles/s41467-021-26938-w#Fig2 in introduction und Figure 2)

**Comment by gordonkoehn** [comment]
ok, will implement as u suggested then.

**Pull request merged**